test/IDE/complete_without_constraint_type.swift

```diff
@@ -1,5 +1,4 @@
 // RUN: %batch-code-completion
 
 struct FileDescriptor: ~#^COMPLETE^# {}
-// FIXME: This should be emitting a `Copyable` declaration instead of a keyword, once there is a declaration for `Copyable`
-// COMPLETE: Keyword/None:                       Copyable; name=Copyable
+// COMPLETE: Decl[Protocol]/OtherModule[Swift]/Flair[RareType]/IsSystem: Copyable[#Copyable#]; name=Copyable
```

Comment:
`Flair[RareType]` is unexpected. Do you know what `CompletionKind` is used for this?

Reply:
Ah, just needed to add `CompletionKind::WithoutConstraintType` to the exclusion list of `CodeCompletionFlairBit::RareTypeAtCurrentPosition`.